kimchi/src/circuits/polynomials/chacha.rs

```diff
@@ -477,7 +477,7 @@ mod tests {
         curve::KimchiCurve,
         proof::{LookupEvaluations, ProofEvaluations},
     };
-    use ark_ff::UniformRand;
+    use ark_ff::{UniformRand, Zero};
```

Comment:
why was it not needed before?

Reply:
I was needed, but it was located near the top of the file.  However, the new version of clippy says it's only needed for the tests module and so it's moved there.